### PR TITLE
initial pull request

### PR DESCRIPTION
**Pull request by HubDub**
this is my pull request for the quiz

### CODE REVIEW

**Comment by HubDub** [comment]
@sscotth I believe I'm ready for you to look at the quiz. Thanks!

**Comment by HubDub** [comment]
@MAOstrander Hi Matt! (is it one T or two? or do you prefer Mathew?). I'm ready for you to look at my quiz when you get a chance! I appreciate it! Debbie

File: index.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+<html>
+<head>
+  <title>My First Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+
+<body>
+<!-- first card box -->
+  <article class="firstcard">
+    <header class="top"><h1>Dull Playing Cards</h1></header>
```
Comment (MAOstrander) [inline index.html]: It isn't a requirement or anything, but typically when you've got nested tags like you space it out to make it a little more readable: 
`<header class="top">
      <h1>Dull Playing Cards</h1>
    </header>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    <section>
+      <img src="images/cards1.jpg">
```
Comment (MAOstrander) [inline index.html]: This totally works, and is fine. Technically the requirements ask for a block level element that contains the image. (which is more of a challenge to do CSS wise) If you feel like revising that, it's decent practice.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      <p>These cards are dull and worn. Broken in. They come in colors. There are 52 of them. They feel good in your hand. </p>
+      <div class="available">Available</div>
+    </section>
+
+    <section>
+      <header class="subheading">Specifications</header>
+      <div class="block">Size: 2" x 3"</div>
+      <div class="block">Weight: 0.5 lb</div>
```
Comment (MAOstrander) [inline index.html:25]: So divs are already block level elements, you don't need to give them a class just to re-add the display block. it doesn't hurt anything, but it isn't needed. ;)

**Comment by MAOstrander** [comment]
👍  Nice use of the figure tag. Not what I expected, but neat!

**Pull request merged**